microlith/prometheus/alerting/couchbase/couchbase-rules.yaml

```
@@ -37,3 +37,21 @@ groups:
         description: The percentage of time that the virtual CPU on node {{ $labels.instance }} is waiting for a real CPU is above 3%.
           This can be due to the fact that a virtual Machine is undersized or the hypervisor is overcommited.
         remediation: Increase the resources available to the VM or reduce the number of VMs on the host machine.
+
+    - alert: CB90043-diskCommitFail
+      expr: |
+        increase(kv_ep_data_write_failed[1m]) > 0
+      for: 0m
+      labels:
+        job: couchbase_prometheus
+        kind: bucket
+        health_check_id: CB90043
+        health_check_name: diskCommitFail
+        cluster: '{{ $labels.cluster }}'
+        node: '{{ $labels.instance }}'
+        bucket: '{{ $labels.bucket }}'
+        severity: critical
+      annotations:
+        title: "Disk Commit Fail on bucket: {{ $labels.bucket }}"
+        description: A disk commit has failed on {{ $labels.bucket }} (cluster {{ $labels.cluster }}, node {{ $labels.hostname }}).
```

```suggestion
        description: A disk commit has failed on {{ $labels.bucket }} (cluster {{ $labels.cluster }}, node {{ $labels.instance }}).
```

`instance` is the label Prometheus sets on anything it scrapes - `hostname` is set using `label_replace` hackery in the other rules (but tbh it's unnecessary there)

---

Worth changing for consistency 👍

---

Not just for consistency, `hostname` won't exist on that metric so there'll be no node :badpokerface: